gists/frontend-pull-request-guidelines.md

```diff
@@ -1,6 +1,23 @@
 # Frontend Pull Request Guidelines
 This document details guidelines for reviewing frontend (especially LPL React-flavored) code. Some of these suggestions may apply more broadly to software development in general, but are mentioned here specifically based on the frequency with which they have been observed in client-side code.
 
+## General
+- Strive for consistency. Review other areas of the code for patterns, copy and paste, make things easier for yourself.
+  - However, don't do it blindly. If you know of a better way, bring it up to the team. Maybe we should change the pattern holistically.
+- Continuously look ahead at tasks/features that are coming up later, and think about how that might affect the task you're working on
+- While comments within the code can be extremely helpful, try not to add them when your code is already clear / self documenting. If you can make some small changes (e.g., variable or function naming) to remove the need for a comment, do that!
+  - And if you make code changes around existing comments, make sure to update those as well to keep them accurate
+- Leave GitHub comments on your own code changes. That forces you to look over all the file changes closely. You might notice something you didn't mean to commit, and you can explain why you made certain decisions to help the reviewer
+- Apply your new learnings to all code changes. When implementing PR feedback, check all of your code changes again to see if the feedback could apply somewhere else
+- Our automatic formatting with Prettier sets tabs equal to 2 spaces. To see this spacing correctly in GitHub, make sure your tab size preference is set to 2 spaces
```

Comment:
I don't think "correctly" is necessary here. One of the major benefits of tabs is that it allows each user to customize it to fit their preference / needs. To say that 2 spaces is "correct" has some implicit associations for users who see better when they take up 4 spaces.